### ISSUE TITLE
Rotating text in the search box is not accessible

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#213]
Rotating text in the search box is not accessibility to AT users. AT user will neverhere the text that keeps change in the form field.

**Comment by ErieMeyer** [comment]
Ack -- good catch.

Suggestion: We should switch static copy to "Search" rather than "Search Data.Gov 2.0" since a) a user won't know what that means (there's no way for a casual visitor to know we just did a relaunch, which is fine), b) Since there will be consistent, iterative releases, no need to have 1.0 2.0 versioning.

![image](https://f.cloud.github.com/assets/1626026/1738062/c0cf7822-637d-11e3-9886-d091615f61fe.png)

**Comment by philipashlock** [comment]
Updated Data.Gov 2.0 to just Data.gov by changing the name of the site in wordpress. 

Added the example search queries to the title attribute as per this suggestion:
http://www.w3.org/TR/2011/WD-html5-20110525/common-input-element-attributes.html#the-placeholder-attribute

Fixed by 3319845b5f3353407049f2758d7bfc07be25c990
